paper/main.tex

```diff
@@ -706,369 +712,343 @@ \section{Evaluation of performance}\label{section:evaluation_of_performance}
-the only features that had a moderate affect are $C_r/C_{\text{mean}},
+In noisy probabilistic ending tournaments and over all the tournaments' results,
+the only features that had a moderate effect are $C_r/C_{\text{mean}},
 C_r/C_{\text{max}}$ and $C_r$. In such environments cooperative behaviour
-appears to be punished by not as much as in noisy and probabilistic ending
+appears to be punished less than in noisy and probabilistic ending
 tournaments.
 
+Moreover, the manner in which a strategy achieves a given cooperation rate
+relative to the tournament population average is important. Playing a strategy
+that randomly cooperates with $C_{\text{mean}}$ is unlikely to be as effective as it will
+randomly have cooperations matched with defections. In contrast, TFT naturally
+achieves a cooperation rate near $C_{\text{mean}}$ by virtue of copying its opponent's
+last move while also minimizing instances where it is exploited by an opponent
+(cooperating while the opponent defects), at least in non-noisy tournaments.
+\footnote{This also explains why Tit For N Tats does not fare well -- it fails
+to achieve the proper cooperation ratio.}
+TFT forces the opponent to pay back the (C, D) round with a (D, C) round before
+returning to mutual cooperation. This explains why TFT performed well
+in Axelrod's original tournaments as most strategies submitted to those tournaments
+were typically cooperative and relatively few strategies used an easily exploitable
+pattern. TFT does not appear in the top ranks in
+these tournaments because it is too nice. Strategies like Grudger will always
+defect after a fixed number of opponent defections, which allows them to
+effectively exploit strategies like Alternator or stochastic strategies that
+have a non-zero chance of cooperating after mutual defection, which TFT will
+not do. Moreover in a noisy environment these strategies will naturally tend
+toward always defecting, leading them to exploit strategies like Cooperator. In
+such a scenario the noisy environment effectively voids Axelrod's rule to be nice,
+allowing strategies to attempt exploitation, whereas in a noise-free environment,
+exploitation is risky because several strategies exhibit a Grudger-like behavior,
+reducing the overall value in attempting to exploit strategies like Cooperator.
+
+Similarly, these results suggest an explanation regarding the perhaps intuitively unexpected
+effectiveness of memory one strategies historically. Given that among the important
+features associated to success are the relative cooperation rate to the population
+average and the four memory-one probabilities of cooperating conditional on the
+previous round of play, all five features can be optimized by a memory one
+strategy such as TFT. Usage of more history becomes valuable when there are
+exploitable opponent patterns, indicated by the importance of SSE as a feature,
+that the first-approximation provided by a memory one strategy is no longer
+sufficient.
+
```

Comment:
I didn't really like the two paragraphs above (lines 729 to 762), I found them quite long winded and tricky to follow. If @Nikoleta-v3 and @marcharper you're both happy with them that's ok with me but if we could perhaps make them a bit more "to the point"? (Perhaps fragmenting long sentences could be sufficient.)